ops/README.md

```diff
@@ -0,0 +1,124 @@
+NEAR application layer running with Tendermint consensus.
+
+## Running locally
+
+To run NEAR locally you would need docker, see [installation instructions](https://www.docker.com/get-started).
+Then run the following:
+```bash
+./ops/deploy_local.sh
+```
+
+After it starts you can open studio at [http://localhost](http://localhost).
+
+To tear it down run:
+```bash
+./ops/teardown_local.sh
+```
+
+## Running remotely
+Similarly you deploy the network to the GCloud:
+```bash
+./ops/deploy_remote.sh
+```
+When the network is deployed it will print the address of the studio.
+
+## Developing locally
+Developing from the source code requires more installation and configuration.
+
+### Installation
+
+This mode of running requires compilation of the Rust code and installed Tendermint.
+
+**1. Dependencies**
+
+Install protobufs:
+```bash
+# Mac OS:
+brew install protobuf
+
+# Ubuntu:
+apt-get install protobuf-compiler
+```
+
+**2. Rust**
+
+Currently, the only way to install NEARMint application layer is to compile it from the source.
+For that we would need to install Rust.
+
+```bash
+curl https://sh.rustup.rs -sSf | sh
+rustup default nightly
+```
+
+
+**3. Source code**
+
+We would need to copy the entire repository:
+
+```bash
+git clone https://github.com/nearprotocol/nearcore
+cd nearcore
+```
+
+**4. Tendermint**
```

Comment:
why not dockerize our setup of tendermint instead of instructions? Instructions seem pretty involved.

Reply:
There is a confusion with what this section is talking about. It is not for dApp developers, but for core developers. The first two sections are for dApp developers.